activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+* Updated `parameterize` to preserve the case of a string, optionally.
+
+    Example:
+
+    parameterize("Donald E. Knuth", separator: '_') # => "donald_e_knuth"
```

Comment:
Sorry, but I forgot to tell you code examples should be indented by three spaces.